universe/jupyterlab/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "packagingVersion": "4.0",
+  "minDcosReleaseVersion": "1.11",
+  "name": "beta-mesosphere-jupyter-service",
+  "version": "1.3.0-0.35.4-beta",
+  "scm": "https://github.com/portworx/dcos-jupyterlab-service/releases/tag/px_1.3-1.3.0-0.35.4",
```

Comment:
Let's not add the portworx's `1.3` version to this because we are not using our dcos commons sdk here. So when you tag keep the version as `px-1.3.0-0.35.4-beta`.
In scm here, you could just point to the repo, no need to add the release tag.